src/shared/components/home/admin-settings.tsx

```diff
@@ -36,6 +37,7 @@ interface AdminSettingsState {
   siteRes: GetSiteResponse;
   siteConfigRes: GetSiteConfigResponse;
   siteConfigHjson: string;
+  legalInfo: string;
```

Comment:
This field needs to be on SiteForm, with the rest of the site fields.

Reply:
Done, doesnt seem to fix anything.

Reply:
No I mean remove the field from here, and move it to SiteForm, along with all the other EditSite columns.